examples/carousel/carousel-2-tablist.html

```diff
@@ -0,0 +1,793 @@
+        </ul>
+      </section>
+
+      <section>
+        <h2 id="ex_label">Example</h2>
+        <div role="separator" id="ex_start_sep" aria-labelledby="ex_start_sep ex_label" aria-label="Start of"></div>
+        <div id="ex1">
+            <section id="myCarousel" class="carousel-tablist" aria-roledescription="carousel" aria-label="Highlighted television shows">
+
+              <div class="carousel-inner">
+
+                <div class="controls">
+
+                  <button class="rotation pause"
+                          aria-label="Stop automatic slide show">
+                    <svg width="42" height="34" version="1.1" xmlns="http://www.w3.org/2000/svg">
+                      <rect class="background" x="2" y="2" rx="5" ry="5" width="38" height="24"/>
+                      <rect class="border" x="4" y="4" rx="5" ry="5" width="34" height="20"/>
+
+                      <polygon class="pause" points="17 8 17 20"/>
+
+                      <polygon class="pause" points="24 8 24 20"/>
+
+                      <polygon class="play" points="15 8 15 20 27 14" />
+
+                    </svg>
+                  </button>
+
+                  <div role="tablist"
+                       aria-label="Slides">
+                    <div class="border">
+                      <button id="carousel-tab-1"
+                              role="tab"
+                              aria-label="Slide 1"
+                              aria-selected="true"
+                              aria-controls="carousel-item-1">
+                        <svg width="34" height="34" version="1.1" xmlns="http://www.w3.org/2000/svg">
+                          <circle class="border" cx="16" cy="15" r="10" />
+                          <circle class="tab-background" cx="16" cy="15" r="8" />
+                          <circle class="tab" cx="16" cy="15" r="6" />
+                        </svg>
+                      </button>
+                      <button id="carousel-tab-2"
+                              role="tab"
+                              tabindex="-1"
+                              aria-label="Slide 2"
+                              aria-selected="false"
+                              aria-controls="carousel-item-2">
+                        <svg width="34" height="34" version="1.1" xmlns="http://www.w3.org/2000/svg">
+                          <circle class="border" cx="16" cy="15" r="10" />
+                          <circle class="tab-background" cx="16" cy="15" r="8" />
+                          <circle class="tab" cx="16" cy="15" r="6" />
+                        </svg>
+                      </button>
+                      <button id="carousel-tab-3"
+                              role="tab"
```

Comment:
Could there be `type="button"` on this and the other carousel buttons?

Reply:
Good suggestion.

Reply:
I will add type=button to the pull request